app/studentsApi.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+let students = require('./studentsDB.json');
+module.exports = (() => {
+
+    return {
+        getStudent: getStudent
+    };
+
+    function getStudent(req, res, next) {
+        let id = req.query.mat;
+        if (id) {
+            let student = students.find(student => student.id === id);
+            student.status = getStatus(student);
+            if (student) {
+                res.status(200).send(student);
+            } else {
+                res.status(200).send({ message: 'Student not found.' });
```

Comment:
When the API is working but we didn't return anything we can use 204 [http status code](http://www.restapitutorial.com/httpstatuscodes.html)